src/MyEventEmitter.js

```diff
@@ -1,12 +1,94 @@
 'use strict';
 
-export class MyEventEmitter {
-  on() {}
-  once() {}
-  off() {}
-  emit() {}
-  prependListener() {}
-  prependOnceListener() {}
-  removeAllListeners() {}
-  listenerCount() {}
+class MyEventEmitter {
+  constructor() {
+    this.events = {
+
+    };
+  }
+
+  addEvent(name, callback, repeat, prepend) {
+    if ({}.hasOwnProperty.call(this.events, name)) {
```

Comment:
Replace `{}.hasOwnProperty.call(this.events, name)` with `Object.prototype.hasOwnProperty.call(this.events, name)` for better readability.

But the idea is really cool!